repo2docker/buildpacks/base.py

```diff
@@ -111,7 +111,15 @@
 
 # Allow target path repo is cloned to be configurable
 ARG REPO_DIR=${HOME}
-ENV REPO_DIR ${REPO_DIR}
+ENV REPO_DIR=${REPO_DIR}
+RUN if [ ! -d "${REPO_DIR}" ] \
+    ; then \
+        mkdir -p "${REPO_DIR}" \
+        && chown -R ${NB_USER}:${NB_USER} "${REPO_DIR}" \
```

Comment:
Perhaps we can remove `-R` as it's an empty directory just created.

Reply:
yeah, it probably doesn't magically do `umask` or anything (though maybe we should think about that)...

Reply:
will wait to get any other feedback, it's a lot of coal to burn!

Reply:
FYI you can use something like `/usr/bin/install -d <directory> -m <mode> -o <owner> -g <group>` to do everything in one go if you want, doesn't really matter though.

Reply:
Yeah, can we use this instead? `if` in bash always makes me feel like I'm too dumb to make it work

Reply:
I'll go for this as resolved, allowing anyone to refactor this further later.

Reply:
Resolved it by applying my own suggestion, based on `/usr/bin/install -d <directory> -m <mode> -o <owner> -g <group>`